src/components/Bar.svelte

```diff
@@ -72,6 +72,12 @@
         await updateSettings(config)
         await updateApps(config)
       }
+      // filter claudy subsection if claudyConfig is null
+      if (!claudyConfig) {
+        if (config.sections && config.sections.drawer) {
+          config.sections.drawer = config.sections.drawer.filter(s => s.length && s[0].slug !== 'claudy')
```

Comment:
Subsections, I used just the initial letter in functions like `sort()` or `filter()` to keep them short if they can be written in one line.